books-api/Book Copies ADR.md

```diff
@@ -136,11 +136,11 @@ erDiagram
     }
     BooksCopiesReadingHistory {
         long id PK
-        long bookCopyId FK
-        long holderEmployeeId
+        long copyId FK
```

Comment:
Let's revert back this field to `bookCopyId `. There was a naming convention, now it is gone.